planning/obstacle_velocity_limiter/src/obstacle_velocity_limiter_node.cpp

```diff
@@ -245,6 +266,35 @@ bool ObstacleVelocityLimiterNode::validInputs(const boost::optional<size_t> & eg
 
   return occupancy_grid_ptr_ && dynamic_obstacles_ptr_ && ego_idx && current_odometry_ptr_;
 }
+
+void ObstacleVelocityLimiterNode::onLanelet2Map(autoware_auto_mapping_msgs::msg::HADMapBin::SharedPtr msg){
+
+    lanelet::utils::conversion::fromBinMsg(*msg, lanelet_map_ptr_);
+
+    if (msg->header.frame_id == "map"){
+        extractStaticObstacles(*lanelet_map_ptr_, obstacle_params_.static_map_tags);
+        return;
+    }
```

Comment:
Huh?

Reply:
I was trying to transform the lanelet2 map into odom frame but it seems to be diffcult. So for right now `extractStaticObstacles` only works if `absolute_odom` is applied to motion planning.

Reply:
Gotcha. Thanks for the explanation.